changelogs/unreleased/su-without-a-function-uid.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* Now using `box.session.su`, both with and without a function, change
```

Comment:
I don't consider this a bug. According to the [documentation](https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_session/su/), calling `box.session.su()` without a function changes the current user permanently, i.e. it should change both `box.session.user()` and `box.session.effective_user()`, while calling `box.session.su()` with a function changes the current user temporarily, i.e. it should only change `box.session.effective_user()`:

<img width="1005" height="387" alt="image" src="https://github.com/user-attachments/assets/695f3c32-43d2-4652-a0ba-45deaf3d5b86" />

Reply:
BTW we're planning to deprecate `box.session.su()` as well as `box.session.effective_user()` soon, see #11690.

Reply:
> I think we can consider [tarantool/tarantool-ee#1278](https://github.com/tarantool/tarantool-ee/issues/1278) a bug.

From the bug description:

> After this action the audit logs will show "user = 'admin'" for all the following actions in this session as if they were triggered by an admin. This can make it hard to investigate security incidents.

There's the `remote` field, which can be used for searching all actions executed in the scope of a particular connection. One would see `box.session.su()` there so I wouldn't agree that changing the current user would complicate investigation of security incidents. Overall, I don't think this issue is worth fixing at all, especially in the light of the future `box.session.su()` deprecation and removal.